app/main.py

```diff
@@ -1,3 +1,17 @@
 class OnlineCourse:
-    # write your code here
-    pass
+
+    def __init__(self, name, description, weeks):
+        self.name = name
+        self.description = description
+        self.weeks = weeks
+
+    @staticmethod
+    def days_to_weeks(days):
+        return (days + 6) // 7
+
+    @classmethod
+    def from_dict(cls, course_dict):
+        cls.name = course_dict["name"]
```

Comment:
`cls` is a reference for `OnlineCourse` class. You shouldn't define its attributes.
Use `cls` as a constructor:
```
return cls(name=..., description=..., weeks=...)
```